README.md

```diff
@@ -38,9 +39,9 @@ Select 1:
 Describe the rules of scope in JavaScript.
 
 Your Answer:
-```text
+```Scope is where a variable can be called. The scope of an element can either be global or local. Global variables are defined outside a function (or inside a function without a var before their name) can be accessed anywhere in the script. Local variables are defined inside functions and only have their set value in the function in which they are defined.
```

Comment:
locally scoped variables are also available within contained functions.  This is sometimes referred to as "chaining of scopes": Any function defined within another function has a local scope which is linked to the outer function.

More info: http://www.coolcoder.in/2014/03/everything-you-need-to-know-about.html